### ISSUE TITLE
Backlinks lists not contain wikilinks after import pod markdown (original wikilinks with Upper case characters)

### ISSUE THREAD

**Issue by voothi** [dendronhq/dendron#1607]
## Please select the area your bug applies to. (Multiple selections are Possible)

- [x] Onboard. Initial vault setup and import/export pods
- [ ] Create. Note creation, lookup, snippets and templates
- [x] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [ ] Misc (Choose this if your not sure)

## Describe the bug

Now backlinks appear only for wikilinks in lower case as a note name, but after import pod markdown my wikilinks remained is original (contain Upper case words). Propose do backlinks register as independent as well as works Direct wikilinks (in Dendron). In Obsidian.md backlinks with my current Dendron multi-vaults is worked correct. 

## To Reproduce

In my vault
1. Create dst note test_note-dendron-imported.md
2. Create src note test-note-new.md; paste wikilinks [[Test_Note-dendron-imported]]
2. Go to dst note test_note-dendron-imported.md 
3. Click on backlinks panel
4. See empty backlinks lists
5. Go to src test-note-new.md; paste wikilinks [[test_note-dendron-imported]]
6. Reply Step 3-4
7. See backlinks to src test-note-new. Worked

## Expected behavior
From ## To Reproduce lists
4. See backlinks to src test-note-new. **Worked**
7. See backlinks to src test-note-new. **Worked**

## Desktop (please complete the following information)

- OS: 
```
OS Name:                   Microsoft Windows Server 2012 R2 Standard
OS Version:                6.3.9600 N/A Build 9600
```
- VSCode
```
Version: 1.58.2
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:10:15.214Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 6.3.9600
```
```
dendron.dendron
v0.65.0
```

## Additional context

https://discord.com/channels/717965437182410783/902566857633062942

## Dendron Log file

None

**Comment by Harshita-mindfire** [comment]
This has been fixed in Dendron 0.66. Please let us know if you are still seeing it, otherwise this issue will automatically close.

**Comment by voothi** [comment]
@Harshita-mindfire Hi!
Thanks!
I will check on test data.
While 66 did not work in my own workspace.

**Comment by Harshita-mindfire** [comment]
@voothi Updated the version to 0.66.1. Thanks!

**Comment by voothi** [comment]
@Harshita-mindfire, hello!
I checked on test data (in the attachment 20211103164759-github-1607.zip).
I did not see the changes.

Video
20211103163313;Dendron;GitHub;1607 https://youtu.be/17WV917tRm4

Vault
[20211103164759-github-1607.zip](https://github.com/dendronhq/dendron/files/7468028/20211103164759-github-1607.zip)

Thanks!

**Comment by voothi** [comment]
Update (fix) files
[20211104132648-github-1607.zip](https://github.com/dendronhq/dendron/files/7474075/20211104132648-github-1607.zip)

**Comment by Harshita-mindfire** [comment]
Hey @voothi, this fix is specific to the import pod's behavior on how it creates the wikilink upon importation, but for the issue around case-sensitive links in general (fname case sensitivity), we're aware of the issue and will be fixed in upcoming releases.

**Comment by voothi** [comment]
@Harshita-mindfire Hi! 
Need some information from me?

**Comment by Harshita-mindfire** [comment]
@voothi Are we good to close this and #848?

**Comment by voothi** [comment]
@Harshita-mindfire @kevinslin 
Hi!
The fact is that does not work now in the version v0.68.1.
It's not about the import mechanism (I will check later), about the view backlinks for current notes.
Currently I don't see backlinks to my old notes if src note contain old wikilinks with Upper-case characters. 
![image](https://user-images.githubusercontent.com/9587340/142047203-0ac42737-8eb9-4b73-aded-c42b024ac4b4.png)
![image](https://user-images.githubusercontent.com/9587340/142047348-aedb305b-4768-42a5-858f-3f63b88e95a2.png)

**Comment by Harshita-mindfire** [comment]
Hi @voothi.
We added a fix in v0.66.1 to update wikilinks as well on import. 
We are also aware of the issue with inconsistency in behaviour of wikilink and backlink when there are mixed case wikilinks. We are tracking it separately #863. I see that your problem coincides with one of the cases described there, let's track this with it and close this one.

**Comment by voothi** [comment]
Hi @Harshita-mindfir 
Yes,  my problem is coincides with #863.
We will track it.
I close this issue.
Thanks!
